examples/TestDriver/src/pages/propExtraction.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
```

Comment:
Is there an exclude case that would be worth testing somewhere in here?

Reply:
Good call - added a test for that